bigquery_etl/public_data/publish_json.py

```diff
@@ -302,6 +302,10 @@ def main():
     if not metadata.is_public_json():
         return
 
+    if metadata.review_bug() is None:
```

Comment:
Would it be possible to call `validate_metadata` here  instead of duplicating some of the validation logic? Or perhaps refactor `validate_metadata` a bit so that we have a common function to call? I want to make sure we perform the same set of checks when running `publish_json` as we do when we run `validate_metadata`.